daemon/lokinet.cpp

```diff
@@ -382,98 +410,83 @@ lokinet_main(int argc, char** argv)
   SetConsoleCtrlHandler(handle_signal_win32, TRUE);
 #endif
 
-  cxxopts::Options options(
-      "lokinet",
-      "LokiNET is a free, open source, private, "
-      "decentralized, \"market based sybil resistant\" "
-      "and IP based onion routing network");
-  // clang-format off
-  options.add_options()
+  CLI::App cli{
+      "LokiNET is a free, open source, private, decentralized, market-based sybil resistand and IP "
+      "based onion routing network",
+      "lokinet"};
+  command_line_options options{};
+
+  // flags: boolean values in command_line_options struct
+  cli.add_flag("--version", options.version, "Lokinet version");
+  cli.add_flag("g,--generate", options.generate, "Generate default configuration and exit");
+  cli.add_flag(
+      "r,--router", options.router, "Run lokinet in routing mode instead of client-only mode");
+  cli.add_flag("f,--force", options.force, "Force writing config even if file exists");
+
+  // options: string
+  cli.add_option("--config", options.configPath, "Path to lokinet.ini configuration file")
+      ->capture_default_str();
+
 #ifdef _WIN32
```

Comment:
could probably use:

```
if constexper(llarp:platform::is_windows)
{
   // ...
}
```

instead of a preprocessor macro.

Reply:
fixed